### ISSUE TITLE
ZORRO ELRS - Default Channel Order ignored in new mixer page

### ISSUE THREAD

**Issue by andrewnewton** [EdgeTX/edgetx#4469]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### What part of EdgeTX is the focus of this bug?

Transmitter firmware

### Current Behavior

Default channel order is set as AETR but new blank mixer page entries appear in RAET order.
No entries in the Inputs page, starting with a blank Mixer page and manually entering new lines.
![P1090276](https://github.com/EdgeTX/edgetx/assets/42705629/7c893ebf-2045-4e53-8db0-8bb32a261a78)
![P1090277](https://github.com/EdgeTX/edgetx/assets/42705629/a5e4ade2-e845-4238-bd5c-7628e5c5c67a)
![P1090275](https://github.com/EdgeTX/edgetx/assets/42705629/77ff6f59-04f7-4126-a16a-6c49757b650a)
![P1090273](https://github.com/EdgeTX/edgetx/assets/42705629/957e1e0c-41c1-4294-9fcd-bd8d74fb3fc9)

### Expected Behavior

Mixer lines should appear in AETR order

### Steps To Reproduce

Delete all INPUT entries and all Mixer lines.
Manually enter mixer lines

### Version

2.9.2

### Transmitter

RadioMaster Zorro

### Operating System (OS)

macOS

### OS Version

Sonoma 14.1.2

**Comment by 3djc** [comment]
No, that's not how it should work. Default channel order is used only for NEWLY created model. Current and existing models will not be affected.

Create a new model, it should be AETR

**Comment by andrewnewton** [comment]
No this is changed from 2.8 
With 2.8 I create a new model
Inputs and Mixer are pre populated in AETR order
Then I delete the pre entered Inputs and Mixer lines
Then when re entering Mixer lines, they appear in AETR order
BUT in 2.9 the reentered mixer lines appear in RAET order

**Comment by pfeerick** [comment]
I can't reproduce this behaviour. On 2.8.4 and 2.9.2 (and a recent 2.10 nightly), the order the first four channels are added on the inputs page is unchanged - it follows the default channel order setting for RAET, TAER and AETR. The mixes page simply adds the same index input (and always has from the looks of the 2.8 code), so is unaffected by the default channel order settings. Since changing the default channel order setting, and then deleting the inputs and readding them changes inputs 1 through 4, the stick assigned to each input will be different, so the mixer line **should** be the unchanged anyway.

**Comment by pfeerick** [comment]
Actually, I just noticed you said with a *blank* inputs page (which is highly unusual btw)... and that seems the be the important factor here:

RETA order on 2.9.2 gives (yes, a double rudder bug there also it seems)
![image](https://github.com/EdgeTX/edgetx/assets/5500713/c8ac6519-e556-49d4-bd8b-7accf5547393)

AETR order on 2.9.2 gives
![image](https://github.com/EdgeTX/edgetx/assets/5500713/16c0d988-56b6-48e9-9848-0eb7b3ef92ee)

RETA order on 2.8.4 gives
![image](https://github.com/EdgeTX/edgetx/assets/5500713/0f846339-c25a-4255-b3e3-e37fa0f26d1b)

AETR order on 2.8.4 gives
![image](https://github.com/EdgeTX/edgetx/assets/5500713/c78958ba-2393-4fb5-98d7-5eb223428053)

**Comment by andrewnewton** [comment]
Thanks for checking this. Here's a video comparing 2.8 with 2.9.2 https://youtu.be/poGl0lLqzVw
